draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/TitleBlockZen.tsx

```diff
@@ -211,11 +214,23 @@ const renderTag = (surveyStatus: SurveyStatus) => {
   )
 }
 
-const renderAvatar = (image: JSX.Element, avatarAutomationId: string) => (
-  <div data-automation-id={avatarAutomationId} className={styles.avatar}>
-    {image}
-  </div>
-)
+const renderAvatar = (
```

Comment:
Maybe you wanna write this like a typeguard? https://www.typescriptlang.org/docs/handbook/advanced-types.html

Reply:
done!